.ibm/pipelines/openshift-tests.sh

```diff
@@ -241,31 +249,60 @@ main() {
   ENCODED_API_SERVER_URL=$(echo "$API_SERVER_URL" | base64)
   ENCODED_CLUSTER_NAME=$(echo "my-cluster" | base64)
 
-  configure_namespace
-  installPipelinesOperator $DIR
+  configure_namespace "$NAME_SPACE"
+  installPipelinesOperator "$DIR"
+  install_helm
+  uninstall_helmchart
+
+  cd "$DIR"
+  apply_yaml_files "$DIR" "$NAME_SPACE"
+  add_helm_repos
+
+  echo "Tag name : ${TAG_NAME}"
+
+  helm upgrade -i "${NAME_SPACE}" -n "${NAME_SPACE}" rhdh-chart/backstage --version "${CHART_VERSION}" -f "$DIR"/value_files/"${HELM_CHART_VALUE_FILE_NAME}" --set global.clusterRouterBase="${K8S_CLUSTER_ROUTER_BASE}" -set upstream.backstage.image.tag=${TAG_NAME}
```

Comment:
Is that intended to use `-i "${NAME_SPACE}"` instead of `-i "${RELEASE_NAME}"`? we may use the same values for the release name and name-space. However, it would be less confusing by using the separate variables at least.

Reply:
@josephca  it's intentional, specially because now we also have helm installation for rbac, it's just to avoid 4 variables only for  it(namespace, namespace_rba, release_name, release_name_rbac)